_posts/2017/2017-10-29-use-simplecov.markdown

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Use simplecov in your Rails test suite'
+description: 'It’s not a silver bullet, but it is useful.'
+layout: article
+category: ruby
+image:
+  base: '2017/use-simplecov'
+  alt: 'Coverage HTML output'
+---
+
+A useful way of assessing the effectiveness of your testing is to use the [`simplecov` gem](https://github.com/colszowka/simplecov) with your Rails application to measure the lines of code that are exercised during your test suite.
+
+## Use…
+
+…`simplecov` in your tests.
+
+#### Add to `Gemfile`
+
+```ruby
+group :test do
+  gem 'simplecov', require: false
+end
+```
+
+And run `bundle` in the root directory of your project
+
+#### Add to `.gitignore`
+
+Do not commit generated files in the `coverage` directory.
+
+```
+coverage/*
+```
+
+#### At the top of `spec/spec_helper.rb` or `test/test_helper.rb`
```

Comment:
The other headings have all been commands: "Use", or "Add to..", etc. This one is different. Perhaps you want to make it the same format, e.g.: "Edit your `spec/spec_helper.rb`..."